roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -69,7 +67,7 @@ RobotCommand ShotController::getRobotCommand(world::Robot robot, const Vector2 &
     }
     else {
         isShooting = false;
-        shotData = goToPlaceBehindBall(robot, lineToDriveOver.first + ball->vel*0.2, lineToDriveOver, currentDesiredGeneva);
+        shotData = goToPlaceBehindBall(robot, lineToDriveOver.first, lineToDriveOver, currentDesiredGeneva);
```

Comment:
Why was this removed? Testing? Seems like the robot will just drive against the ball now when it is moving away from the robot which is not behavior you really want

Reply:
this uses ballhandle now, which automatically handles everything with the ball for you.
this means that you do not have to take anything into account anymore, so also not the ball velocity.